sample-skill/lambda/lambda_function.py

```diff
@@ -254,7 +278,12 @@ def can_handle(self, handler_input: HandlerInput) -> bool:
 
     def handle(self, handler_input: HandlerInput) -> Response:
 
-        speak_output = get_speak_output_get_course_progress(handler_input)
+        # Change the email auth backend to use a different authentication method.
+        email_auth_backend = AlexaEmailAuthentication(handler_input)
```

Comment:
can use use a .env file instead to configure which backend to use?

Reply:
We could use the alexa one as the default, but configurable by changing the .env file instead of the code